letsencrypt/client.py

```diff
@@ -125,6 +130,28 @@ def register(config, account_storage, tos_cb=None):
 
     return acc, acme
 
+def perform_registration(acme, config):
+    """
+    Actually register new account, trying repeatedly if there are email
+    problems
+
+    :param .IConfig config: Client configuration.
+    :param acme.client.Client client: ACME client object.
+
+    :returns: Registration Resource.
+    :rtype: `acme.messages.RegistrationResource`
+
+    :raises .UnexpectedUpdate:
+    """
+    try:
+        return acme.register(messages.NewRegistration.from_data(email=config.email))
+    except messages.Error, e:
```

Comment:
Yep. I'm sure. [Proof](https://travis-ci.org/letsencrypt/letsencrypt/jobs/92367746#L751).

I'll comment on the tests about the problem.